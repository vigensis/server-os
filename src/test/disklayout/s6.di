SCSI	disk0	MFG	SSD	600127266816	no	yes
SCSI	disk1	MFG	SSD	600127266816	no	yes
SCSI	disk2	MFG	SSD	600127266816	no	yes
SCSI	disk3	MFG	SSD	600127266816	no	yes
SCSI	disk4	MFG	SSD	600127266816	no	yes
SCSI	disk5	MFG	SSD	600127266816	no	yes
USB	key0	USBMFG	USB Key	15737028608	yes	no
