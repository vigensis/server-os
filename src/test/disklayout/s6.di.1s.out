{
	"spares": [
		{
			"name": "disk5",
			"vid": "MFG",
			"pid": "SSD",
			"size": "600127266816",
			"solid_state": true
		}
	],
	"vdevs": [
		{
			"type": "raidz",
			"devices": [
				{
					"name": "disk0",
					"vid": "MFG",
					"pid": "SSD",
					"size": "600127266816",
					"solid_state": true
				},
				{
					"name": "disk1",
					"vid": "MFG",
					"pid": "SSD",
					"size": "600127266816",
					"solid_state": true
				},
				{
					"name": "disk2",
					"vid": "MFG",
					"pid": "SSD",
					"size": "600127266816",
					"solid_state": true
				},
				{
					"name": "disk3",
					"vid": "MFG",
					"pid": "SSD",
					"size": "600127266816",
					"solid_state": true
				},
				{
					"name": "disk4",
					"vid": "MFG",
					"pid": "SSD",
					"size": "600127266816",
					"solid_state": true
				}
			]
		}
	],
	"capacity": 2400509067264
}
